src/panel-triggers/components/AlertList/AlertList.tsx

```diff
@@ -206,6 +234,18 @@ export default class AlertList extends PureComponent<AlertListProps, AlertListSt
               {`${texts.hideAlertsInMaintenance} ${amountOfAlertsInMaintenance}${texts.pieces}`}
             </label>
           </div>
+          <div className={'checkbox-filter ' + (amountOfAlertsWithTestTag === 0 ? 'disabled' : '')}>
+            <input
+              type="checkbox"
+              id="showTestTags"
```

Comment:
Id could also be hide instead of show ..

Reply:
Yup.

Reply:
In commit 02164ba2f0c471e0b8c76d00329f640da08af9e2.